src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -775,6 +783,9 @@ async def _genomic_to_tx_segment(
                 return GenomicTxSeg(
                     errors=[f"Genomic accession does not exist in UTA: {genomic_ac}"]
                 )
+            if starting_assembly == Assembly.GRCH37:
+                grch38_ac = await self.uta_db.get_newest_assembly_ac(genomic_ac)
+                genomic_ac = grch38_ac[0]
```

Comment:
I'm kind of wondering if we need to do the above validation if we have this?

Reply:
I think this step is needed so we can convert the GRCh37 genomic accession to the GRCh38 genomic accession

Reply:
I'm talking about if lines 781-785 are still needed since you added lines 786-788 (I don't think so, but not sure if I'm missing something):

```python
            genomic_ac_validation = await self.uta_db.validate_genomic_ac(genomic_ac)
            if not genomic_ac_validation:
                return GenomicTxSeg(
                    errors=[f"Genomic accession does not exist in UTA: {genomic_ac}"]
                )
```

Reply:
Ah I see. Yes I think those lines can be removed. We can validate if a genomic accession exists by seeing if `get_newest_assembly_ac` does not return an empty list

Reply:
I looked into this and I think we should keep lines 781-785 in. This function is needed to test if an accession like `NC_000001.200` is invalid since it does not exist in UTA

Reply:
It'd only be necessary for GRCh38 starting assembly though, right? Since `await self.uta_db.get_newest_assembly_ac(genomic_ac)` would not return anything if it's an invalid accession?

Reply:
Oh, I think `get_newest_assembly` removes the version

Reply:
Is it wrong to do this if they are still talking about chromosome 1?